include/inexor/vulkan-renderer/renderer.hpp

```diff
@@ -73,7 +73,7 @@ class VulkanRenderer {
     std::unique_ptr<wrapper::GLFWContext> m_glfw_context;
     std::unique_ptr<wrapper::Window> m_window;
     std::unique_ptr<wrapper::Instance> m_vkinstance;
```

Comment:
We should probably rename this from `m_vkinstance` to `m_instance` too.

Reply:
Done.